packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -287,6 +360,18 @@ export const getActivitiesByAddress = async function (
     activityCache.set(ckActivities, preProcessedActivities);
   }
 
+  // Fetch token prices for fiat value calculations
+  let tokenPrices: Array<{ symbol: string; prices: Record<string, number> }> =
+    [];
+  try {
+    tokenPrices = await Bridge.tokens({
```

Comment:
Important: We should avoid direct invocation of `Bridge.tokens` here. This will result in tight coupling with Thirdweb.

Reply:
we should use `getTokenFiatPrices`